lib/websocket.js

```diff
@@ -30,13 +30,17 @@ const closeTimeout = 30 * 1000;
 /**
  * Class representing a WebSocket.
  *
+ * @property {Number} CONNECTING The connection is not yet open
+ * @property {Number} OPEN The connection is open and ready to communicate
+ * @property {Number} CLOSING The connection is in the process of closing
+ * @property {Number} CLOSED The connection is closed
```

Comment:
tried this also, didn't work for me, maybe it's the right way to do it...

what did work was:
```js
class X {
  static CONNECTING = 1
  CONNECTING = 1
}
```
or like a mention earlier: `x.CONNECTING = 1`
but then it dose not have the right descriptor (unless you add those with defineProperties)

Reply:
Does it work with

```js
Object.defineProperty(WebSocket.prototype, 'CONNECTING', { enumerable: true, value: 0 });
Object.defineProperty(WebSocket, 'CONNECTING', { enumerable: true, value: 0 });
```

(without the loop)?

Reply:
The same pattern is also used here https://github.com/websockets/ws/blob/7.5.2/lib/websocket.js#L402-L411. I'm not sure I want to change it. It is not a JSDoc issue.

Reply:
without the loop it did work fine doing just
```js
Object.defineProperty(WebSocket.prototype, 'CONNECTING', { enumerable: true, value: 0 });
```

tried it if it would work with a object using defineProperties


```js
Object.defineProperties(WebSocket, {
  CONNECTING: {
    enumerable: true,
    value: 0
  }
});
```
The result: `Property 'CONNECTING' does not exist on type 'typeof WebSocket'`